skyportal/handlers/api/group_admission_request.py

```diff
@@ -205,8 +186,6 @@ def post(self):
             )
 
         self.push(action="skyportal/FETCH_USER_PROFILE")
```

Comment:
I'm not sure why there is `self.push(action="skyportal/FETCH_USER_PROFILE")` in the first place.

Reply:
I wouldn't be surprised if it was put into there by accident ;)